tests/Traits/TerminusTestTrait.php

```diff
@@ -22,7 +22,7 @@ trait TerminusTestTrait
      */
     protected static function callTerminus(string $command, ?string $pipeInput = null): array
     {
-        $procCommand = sprintf('%s %s', TERMINUS_BIN_FILE, $command);
+        $procCommand = sprintf('%s %s', getenv('TERMINUS_BIN_FILE'), $command);
```

Comment:
I think this should stay as it was. I mean, we always want to run tests against the latest phar so not sure why it would make sense to do this change

Reply:
It's possible you might want to test against an earlier version.  Not always the best policy to hardcode. :)